NEWS.md

```diff
@@ -10,6 +10,9 @@ User-visible changes worth mentioning.
   the /oauth/token path. Switch from using a DateTime object to update
   AR to using a Time object. (Issue #668)
 - [#677] Support editing application-specific scopes via the standard forms.
+- [#678] Change application-specific scopes to take precedence over server-wide
+  scopes. This removes the previous behavior where the intersection between
+  application and server scopes was used.
```

Comment:
This is super important (and so much better)! Can you please add it at the top, in a subsection `### Backward incompatible changes`?